docs/uncertainty/index.rst

```diff
@@ -166,14 +171,18 @@ Additionally, Poisson and uniform |distribution| creation functions exist::
                [...],
                [...]] kpc with n_samples=1000>
 
+.. EXAMPLE END
+
 Users are free to create their own distribution classes following similar
 patterns.
 
-
 Using Distributions
 -------------------
 
-This object now acts much like a |quantity| or |ndarray| for all but the
+.. EXAMPLE START: Using and Accessing Distributions
+.. not quite sure what to call this example or how best to break it up

```

Comment:
Maybe `[Accessing] Properties of Distributions`? I think I would not break it up further.

Reply:
Good to know and yes that title is better!